src/window_identifier/mod.rs

```diff
@@ -310,6 +314,47 @@ impl Serialize for WindowIdentifierType {
     }
 }
 
+impl WindowIdentifierType {
+    /// Sets the given window as a modal child of the window represented by
+    /// self. Currently only Wayland windows are supported.
+    ///
+    /// This is useful in backend implementations as the portal dialogs have to
+    /// be modal and grouped together with the application that launched the
+    /// request.
+    ///
+    /// Realizes the window if it is not realized yet.
+    ///
+    /// Returns `true` on success.
+    #[allow(unused_variables)]
+    #[cfg(any(feature = "gtk4_wayland", feature = "gtk4_x11"))]
```

Comment:
```suggestion
    #[cfg(all(any(feature = "gtk4_wayland", feature = "gtk4_x11")), feature = "backend")]
```
to avoid it being exposed all the time